toolchain/codegen/codegen.h

```diff
@@ -0,0 +1,20 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_CODEGEN_CODEGEN_H_
+#define CARBON_TOOLCHAIN_CODEGEN_CODEGEN_H_
+
+#include <cstdint>
+
+#include "llvm/IR/Module.h"
+
+namespace Carbon {
+
+class CodeGen {
+ public:
+  llvm::raw_ostream& output = llvm::outs();
+  void GenerateObjCodeFromModule(llvm::Module& module);
+};
```

Comment:
```suggestion
void GenerateObjCodeFromModule(llvm::Module& module);
```

We prefer free functions when the class isn't used.

Also, please add comments explaining what this does.